main.py

```diff
@@ -185,3 +188,31 @@ def get_cookie(dpid):
         """Get 8-byte integer cookie."""
         int_dpid = int(dpid.replace(":", ""), 16)
         return (0x00FFFFFFFFFFFFFF & int_dpid) | (settings.COOKIE_PREFIX << 56)
+
+    def set_flow_table_group_owner(self, flow: dict, group: str) -> dict:
```

Comment:
Looks good to also have the option to pass the `group: str` here and not always derive from the flow (for NApps that only have a single table group). For NApps like coloring that will only have the `base` group table so far, you could also set `group="base"` in the method parameter. Can you update this?

Reply:
Added default value `"base"` to argument